pkg/reconciler/kubernetes/tektonpipeline/tektonpipeline.go

```diff
@@ -373,6 +384,25 @@ func (r *Reconciler) targetNamespaceCheck(ctx context.Context, tp *v1alpha1.Tekt
 	return err
 }
 
+func (r *Reconciler) createTargetNamespace(ctx context.Context, labels map[string]string, tp *v1alpha1.TektonPipeline) error {
```

Comment:
🧑‍💻 👍  that we can pass labels map as an argument.

so shall we reuse this function in TektonConfig Reconciler as well. ([here](https://github.com/tektoncd/operator/blob/dd94c9dd26ff852cb2fc4dcaf64288ee5f8c856a/pkg/reconciler/shared/tektonconfig/tektonconfig.go#L207))